ui/DesignSystem/Modal.svelte

```diff
@@ -6,8 +6,12 @@
  LICENSE file.
 -->
 <script lang="typescript">
+  import { Emoji } from "ui/DesignSystem";
   export let dataCy: string = "";
   export let style: string = "";
+  export let emoji: string = "";
```

Comment:
I'd prefer if we'd type props like this:

```suggestion
  export let emoji: string | undefined = undefined
```